dl.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -e
 
-VER="disco"
+VER="eoan"
```

Comment:
If this changed then need update more files,
or load `libjavascriptcoregtk` with changed VER after others

Reply:
Sent another PR to update from disco to eoan.

Reply:
Rebased this commit since the disco->eoan change is in.